### PR TITLE
chore(GitHub Actions): add `electron.yml` workflow

### PR DESCRIPTION
**Pull request by bludnic**
Add GitHub Workflows for building the Electron app for each platform:

- electron-mac.yml
- electron-linux.yml
- electron-windows.yml

https://trello.com/c/aQ85UVMM/332-add-github-workflow-for-electron-apps

### CODE REVIEW

File: build/osx/entitlements.mac.plist [hunk 1]
```diff
@@ -6,5 +6,7 @@
     <true/>
     <key>com.apple.security.device.camera</key>
     <true/>
+    <key>com.apple.security.cs.allow-jit</key>
```
Comment (adamant-al) [inline build/osx/entitlements.mac.plist:9]: What's the reason to allow jit?
https://developer.apple.com/documentation/bundleresources/entitlements/com_apple_security_cs_allow-jit

Earlier, builds worked without the this key.

File: notarize.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  }
+
+  if (process.env.SKIP_NOTARIZATION === 'true') {
+    console.log('SKIP_NOTARIZATION=true | Skipping the notarization"')
+    return
+  }
+
+  const appName = context.packager.appInfo.productFilename
+
+  return await notarize({
```
Comment (martiliones) [inline notarize.js:18]: remove `async` from function definition and return raw promise

**Review by martiliones — CHANGES_REQUESTED** [review]

File: build/osx/entitlements.mac.plist (hunk 1 above)
Comment (bludnic) [reply build/osx/entitlements.mac.plist:9]: This is required for notarization.
https://kilianvalkhof.com/2019/electron/notarizing-your-electron-application/

> One of the things Apple snuck in when adding the **notarization is the requirement of a hardened runtime** for your app, basically giving your application less rights by default.

> If you run a hardened runtime, it’s important to set at least two entitlements, “allow-unsigned-executable-memory” and **“allow-jit”**.

File: notarize.js (hunk 2 above)
Comment (bludnic) [reply notarize.js:18]: Removed "await". I'll keep "async" to make it clear that the function must return a promise.

File: .github/workflows/electron-linux.yml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository 🛎️
+        uses: actions/checkout@v3
+
+      - name: Setup Node version 🍀
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
```
[the remark this answers is not in the document]
Comment (bludnic) [reply .github/workflows/electron-linux.yml:23]: I'll leave v16 for now

File: notarize.js (hunk 2 above)
Comment (martiliones) [reply notarize.js]: why does it need to return a promise?

File: build/osx/entitlements.mac.plist (hunk 1 above)
Comment (bludnic) [reply build/osx/entitlements.mac.plist]: Removed "allow-jit". The notarization works as well.

**Review by joebrock666 — APPROVED** [review]

**Review by adamant-al — APPROVED** [review]

**Pull request merged**